mumd/src/network/udp.rs

```diff
@@ -179,28 +185,19 @@ async fn listen(
 }
 
 async fn send_ping(sink: &mut UdpSender, server_addr: SocketAddr) {
```

Comment:
We should send periodic ping packets, since that's how we determine whether or not to tunnel audio through TCP.